docs/stories/stories.md

```diff
@@ -0,0 +1,79 @@
+# COGS User Stories
+
+## Roles:
+User
+Project owner
+Principal Investigator
+Project Authoriser
+Attribution Authoriser
+RSE
+System Administrator
+
+## Stories
+### Project creation
+- Users with valid, non-student institutional email addresses can create new projects as the project owner
+- Users with student or non-institutional email address cannot create new projects
+- New projects are created with a default priority level and are not active
```

Comment:
New projects are created with the 'Awaiting Approval' status. The project must be approved by a member of staff in order to become active.